azurerm/resource_arm_dns_zone.go

```diff
@@ -45,6 +45,29 @@ func resourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
+				Type:     schema.TypeString,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+			},
+
+			"registration_virtual_networks": {
+				Type:     schema.TypeList,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			"resolution_virtual_networks": {
+				Type:     schema.TypeList,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
```

Comment:
(as above) can we remove the default / computed here and make this `resolution_virtual_network_ids`

Reply:
done.